leku/src/main/java/com/adevinta/leku/geocoder/AndroidGeocoderDataSource.kt

```diff
@@ -3,42 +3,35 @@ package com.adevinta.leku.geocoder
 import android.location.Address
 import android.location.Geocoder
 import com.google.android.gms.maps.model.LatLng
-import io.reactivex.rxjava3.core.Single
 import java.io.IOException
 
 private const val MAX_RESULTS = 5
 
 class AndroidGeocoderDataSource(private val geocoder: Geocoder) : GeocoderDataSourceInterface {
 
-    override fun getFromLocationName(query: String): Single<List<Address>> {
-        return Single.fromCallable {
-            try {
-                geocoder.getFromLocationName(query, MAX_RESULTS)
-            } catch (exception: IOException) {
-                emptyList()
-            }
+    override fun getFromLocationName(query: String): List<Address> {
```

Comment:
geocoder.getFromLocationName(query, MAX_RESULTS) can block the thread? should `getFromLocationName` be a suspend fun?

Reply:
I'm not sure about this, I think It should not be a problem because is always called under a suspend fun. And I also think that has a time out exception that raises the IOException.